Makefile

```diff
@@ -278,6 +287,12 @@ generate-go-kubeadm-control-plane: $(CONTROLLER_GEN) $(CONVERSION_GEN) ## Runs G
 		--output-file-base=zz_generated.conversion \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
 
+.PHONY: generate-go-operator
+generate-go-operator: $(CONTROLLER_GEN) ## Runs Go related generate targets for the operator
```

Comment:
Running this updates a line in `exp/operator/api/v1alpha4/zz_generated.deepcopy.go` - pretty minor (just a comment) but would be nice to rerun in this PR.